test/jdk/java/nio/file/WatchService/SensitivityModifier.java

```diff
@@ -93,51 +91,6 @@ static void doTest(Path top) throws Exception {
 
             // register the directories (random sensitivity)
             register(dirs, watcher);
-
```

Comment:
I assume there is a lot more that is no longer needed in this test, e.g most of doTest is not needed, POLL_TIMEOUT_SECONDS can be removed, ...  I suspect the resulting test will be very simple - just test that the temp dir can be registered with each of the sensitivity levels.

Reply:
Fixed (enough?) in 2e9867bd45cfbad727699a1254514227b995dd7a.